src/schema/v2/__tests__/auction_result.test.ts

```diff
@@ -28,9 +28,6 @@ const mockAuctionResult = {
     centsUSD: 200000,
     display: "€200.000",
   },
-  estimate: {
```

Comment:
This was here but it wasn't needed for the resolver since we are determining the estimate using `low_estimate_cents` and `high_estimate_cents`